### DISCUSSION TITLE
Sov's List - Grants

### DISCUSSION THREAD

**Discussion by 0x4007** [ubiquity/business-development#14] [Ideas]
https://blockworks.co/grants

Filtered individual development grants for Ethereum ecosystem and found:

- Compound
- Aave
- "Ethereum ESP" not sure what that is
- dYdX

Otherwise you can search for the [list of companies that offer grant programs](https://blockworks.co/grants/programs) (this is good to stay on top of.)

[Telegram chat for "Crypto Grants Wire"](https://t.me/cryptograntwire)

More context on [Sov's site](https://sovs.notion.site/Sov-s-Compendium-41f097d28dae4d09801f10cde1b2d03b).

**Comment by 0x4007** [comment]
@Hodlatoor This is something for you to look into.

**Reply by yulian228** [reply]
nicely done!

**Reply by Hamilt18** [reply]
agree

**Comment by kOLIANIKO** [comment]
This is the best application in  Crypto

**Comment by Kivamar** [comment]
Good grants for good products

**Comment by cryptochenko** [comment]
interesting

**Comment by Hamilt18** [comment]
not

**Comment by Nisogok** [comment]
Great product.It's a pleasure to use the site.Good grants
